src/Command/Serve.php

```diff
@@ -103,7 +119,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return ExitCode::OK;
         }
 
-        passthru('"' . PHP_BINARY . '"' . " -S $address -t \"$documentRoot\" $router");
+        passthru('"' . PHP_BINARY . '"' . " -S $address -t $documentRoot $router");
```

Comment:
Can there be problems if `$documentRoot` contains spaces?

Reply:
Good catch.